### ISSUE TITLE
Add -h/--help to command-line

### ISSUE THREAD

**Issue by fracpete** [waikato-ufdl/wai-annotations#16]
Add `-h/--help` to the convert-annotations tool, which outputs a short help screen:
```
Use option --list-plugins to list all plugins and their associated options.
See following URLs for usage and examples:
https://github.com/waikato-ufdl/wai-annotations/blob/master/doc/USAGE.md
https://github.com/waikato-ufdl/wai-annotations/blob/master/doc/EXAMPLES.md
```
The above output will be more helpful than the current output when trying `convert-annotations --help`:
```
Traceback (most recent call last):
  File "/home/fracpete/development/projects/waikato-ufdl/wai-annotations.venv/lib/python3.7/site-packages/wai/annotations/main/_main.py", line 67, in sys_main
    main()
  File "/home/fracpete/development/projects/waikato-ufdl/wai-annotations.venv/lib/python3.7/site-packages/wai/annotations/main/_main.py", line 54, in main
    conversion_chain.save()
  File "/home/fracpete/development/projects/waikato-ufdl/wai-annotations.venv/lib/python3.7/site-packages/wai/annotations/core/chain/_ConversionChain.py", line 238, in save
    raise ConversionChainHasNoWriter()
wai.annotations.core.error._ConversionChainHasNoWriter.ConversionChainHasNoWriter
```

**Comment by csterling** [comment]
Additional TODO:
- change `--list-plugins` option to be a simple list of available plugin keywords
- add `--help-plugins` to do what `--list-plugins` does now (usage on all plugins)
- allow `--help-plugins` to take a plugin name and only show usage for that plugin

**Comment by csterling** [comment]
Currently, command-line parsing is done by manually splitting the arguments around known plugin names. If plugin names are to be given to a `--help-plugins` option, this method of splitting will need to be modified.

**Comment by csterling** [comment]
This functionality is implemented in commit 65375f14f3bbe5581b8c2305faed736ee81f88cc
